cfn-resources/project-ip-access-list/mongodb-atlas-projectipaccesslist.json

```diff
@@ -77,42 +83,16 @@
     "/properties/ProjectId",
     "/properties/Profile"
   ],
-  "properties": {
-    "AccessList": {
-      "items": {
-        "$ref": "#/definitions/accessListDefinition"
-      },
-      "type": "array",
-      "insertionOrder": false
-    },
-    "ProjectId": {
-      "description": "Unique 24-hexadecimal digit string that identifies your project.",
-      "type": "string"
-    },
-    "ListOptions": {
-      "$ref": "#/definitions/listOptions"
-    },
-    "TotalCount": {
-      "description": "Number of documents returned in this response.",
-      "type": "integer"
-    },
-    "Profile": {
-      "type": "string",
-      "description": "Profile used to provide credentials information, (a secret with the cfn/atlas/profile/{Profile}, is required), if not provided default is used",
-      "default": "default"
-    }
-  },
   "readOnlyProperties": [
-    "/properties/AccessList",
-    "/properties/AccessList/AwsSecurityGroup"
+    "/properties/TotalCount"
   ],
   "createOnlyProperties": [
     "/properties/ProjectId",
-    "/properties/Profile"
+    "/properties/Profile",
+    "/properties/AccessList"
   ],
   "required": [
-    "ProjectId"
-  ],
-  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
-  "typeName": "MongoDB::Atlas::ProjectIpAccessList"
+    "ProjectId",
+    "AccessList"
```

Comment:
if the user does not provide access-list entries, there is no use for this resource.